posthog/api/insight.py

```diff
@@ -303,6 +309,32 @@ def trend(self, request: request.Request, *args: Any, **kwargs: Any) -> Response
             if len(result["result"]) >= BREAKDOWN_VALUES_LIMIT
             else None
         )
+        if self.request.accepted_renderer.format == "csv":
+            csvexport = []
+            for item in result["result"]:
+                line = {"series": item["label"]}
+                total_sum = 0
+                for index, data in enumerate(item["data"]):
+                    line[item["labels"][index]] = data
+                    total_sum += data
+
+                line["total sum"] = total_sum
+                csvexport.append(line)
+            renderer = csvrenderers.CSVRenderer()
+            renderer.header = csvexport[0].keys()
+            export = renderer.render(csvexport)
+            if request.GET.get("export_insight_id"):
+                export = "{}/insights/{}/\n".format(SITE_URL, request.GET["export_insight_id"]).encode() + export
+
+            response = HttpResponse(export)
+            response[
+                "Content-Disposition"
+            ] = 'attachment; filename="posthog - {name} ({date_from} - {date_to}).csv"'.format(
```

Comment:
Proposal for a subjectively slightly nicer/more helpful filename (putting "PostHog" at the end):
```suggestion
            ] = 'attachment; filename="{name} ({date_from}-{date_to}) from PostHog.csv"'.format(
```